rafthttp/stream.go

```diff
@@ -380,6 +400,19 @@ func (cr *streamReader) dial() (io.ReadCloser, error) {
 		return nil, err
 	case http.StatusOK:
 		return resp.Body, nil
+	case http.StatusNotFound:
+		b, err := ioutil.ReadAll(resp.Body)
+		resp.Body.Close()
+		if err != nil {
+			return nil, err
+		}
+		// It happens only when it connects to etcd 2.0 and etcd 2.0
+		// doesn't understand the path.
+		// TODO: etcd 2.1 should return StatusBadRequest in this case.
```

Comment:
It is covered in 'rafthttp: return StatusBadRequest when stream handler cannot understand'. i am gonna remove this line.